api/src/wfl/api/handlers.clj

```diff
@@ -62,17 +62,17 @@
            succeed))))
 
 (defn post-create
-  "Create the workload described in BODY of REQUEST."
-  [{:keys [parameters] :as request}]
-  (let [{:keys [body]} parameters
-        {:keys [email]} (gcs/userinfo request)]
-    (logr/infof "post-create endpoint called: body=%s" body)
+  "Create the workload described in REQUEST."
+  [request]
+  (let [workload-request (-> (:body-params request)
+                             (rename-keys {:cromwell :executor}))
```

Comment:
interesting, now I almost feel like we want to put this renaming in a middleware :P (maybe too crazy?)

Reply:
:runs screaming:

Reply:
No. This renaming is a workaround for API compatibility.